easybuild/easyconfigs/m/MATLAB/MATLAB-R2018b.eb

```diff
@@ -0,0 +1,23 @@
+name = 'MATLAB'
+version = 'R2018b'
```

Comment:
@vanzod In previous `MATLAB` easyconfig files, we've never used the `R` in the version, no need to start doing that?

Reply:
Fixed